app/javascript/ReadOnlyControl.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class ReadOnlyControl extends React.Component {
+  render() {
+    const { value } = this.props;
+    return <p>{value.toString()}</p>;
+  }
+}
+
```

Comment:
I think this would more idiomatically be a functional component:

```suggestion
export default function ReadOnlyControl({value}) {
  return <p>{value.toString()}</p>;
}
```